### PR TITLE
Updating portal and wts in qa-brh

### PR DESCRIPTION
**Pull request by nss10**
Link to Jira ticket if there is one:

### Environments
- qa-brh

### Description of changes
- Update data-portal to 5.0.0
- Set wts to 0.4.1
- Set Kayako-wrapper-service to 0.4.1

### CODE REVIEW

File: qa-brh.planx-pla.net/manifest.json
Comment (aartivnkt) [inline qa-brh.planx-pla.net/manifest.json:27]: LGTM. The build is currently failing for 0.4.1 (https://github.com/uc-cdis/workspace-token-service/pull/47). As soon as that's working we can merge this in

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- metadata

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- metadata

File: qa-brh.planx-pla.net/manifest.json
Comment (aartivnkt) [reply qa-brh.planx-pla.net/manifest.json:27]: This new 0.4.1 has wts deployment errors in the pod per Jenkins tests, some `ImagePullBackOff` error. Can you look at the logs in qa? Happy to look at it together, we can post in pe-on-call ask for some more guidance.

**Review by aartivnkt — APPROVED** [review]
LGTM, we can merge this in after jenkins build passes!

**Pull request merged**
